index.js

```diff
@@ -1,6 +1,6 @@
-import { DateTime } from './modules/luxon.js';
-import Book from './modules/user.js';
-import navBar from './modules/navigation.js';
+import { DateTime } from './module/luxon.js';
```

Comment:
- [ ] Please make sure that the time gets displayed on the screen as per the requirements.